static/js/settings_playgrounds.js

```diff
@@ -88,6 +90,23 @@ function build_page() {
     meta.loaded = true;
     populate_playgrounds(page_params.realm_playgrounds);
 
+    $(".admin_playgrounds_table").on("click", ".delete", function (e) {
+        e.preventDefault();
+        e.stopPropagation();
+        const btn = $(this);
+
+        channel.del({
+            url: "/json/realm/playgrounds/" + encodeURIComponent(btn.attr("data-playground-id")),
+            error(xhr) {
+                ui_report.generic_row_button_error(xhr, btn);
+            },
+            success() {
+                const row = btn.parents("tr");
+                row.remove();
+            },
```

Comment:
This success function should do nothing -- the row should be removed by the `events` handler.